apiserver/admin.go

```diff
@@ -521,7 +518,11 @@ func (f modelUserEntityFinder) FindEntity(tag names.Tag) (state.Entity, error) {
 		return f.st.FindEntity(tag)
 	}
 
-	modelUser, err := f.st.UserAccess(utag, f.st.ModelTag())
+	model, err := f.st.Model()
```

Comment:
To confirm - eventually, this entity finder will be given a model and not a state right? So this line will be unnecessary.
Maybe we should comment func (st *State) Model() with a todo saying it is deprecated.